api/v1alpha3/awscluster_types.go

```diff
@@ -146,9 +146,9 @@ type Bastion struct {
 
 // AWSLoadBalancerSpec defines the desired state of an AWS load balancer.
 type AWSLoadBalancerSpec struct {
-	// Scheme sets the scheme of the load balancer (defaults to internet-facing)
-	// +kubebuilder:default=internet-facing
-	// +kubebuilder:validation:Enum=internet-facing;internal
+	// Scheme sets the scheme of the load balancer (defaults to Internet-facing)
+	// +kubebuilder:default=Internet-facing
+	// +kubebuilder:validation:Enum=Internet-facing;internal
```

Comment:
```suggestion
	// Scheme sets the scheme of the load balancer (defaults to Internet-facing)
	// +kubebuilder:default=internet-facing
	// +kubebuilder:validation:Enum=internet-facing;Internet-facing;internal
```

Also thinking about the backport here, and I think this is what we'll be accepting?

Reply:
We also want to fix the v1alpha3 branch though, which means allowing lower case `internet-facing`. Let's discuss this when you're around.